tests/gpu_operator_metrics_test.go

```diff
@@ -25,35 +29,68 @@ var _ = Describe("test_gpu_operator_metrics :", Ordered, func() {
 		kubeconfig        string
 		namespace         string
 		dcgmPodServerPort string
+		gpuOpVersion      *version.OperatorVersion
+		monitoringLabel   string
 	)
 
 	BeforeAll(func() {
 		kubeconfig = internal.Config.KubeconfigPath
 		dcgmPodServerPort = "9400"
+		monitoringLabel = "openshift.io/cluster-monitoring"
```

Comment:
Yes 100%. 
Though…. I would like to address this in my next PR that will also modify this file, by adding more metric tests. 
Since this PR was running too many CI runs, I prefer letting this one go, And not waste more resources just for this minor change that will be coming soon enough 😉. 
But you are indeed right. 

Do you agree?